DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -2025,6 +2025,11 @@ globus_l_gfs_iRODS_recv(
     collection = strdup(transfer_info->pathname);
     iRODS_l_reduce_path(collection);
 
+    irods::at_scope_exit free_collection{[&collection] {
+            std::free(collection);
+            collection = nullptr;
```

Comment:
Indentation.

Reply:
My vim setup does that because of the two curly brackets.  I fixed it, squashed, and pounded.